doc/design/send_recv.md

```diff
@@ -0,0 +1,68 @@
+# Send and Recv OP
+
+With PaddlePaddle Fluid, the node to node communication is done
+through the send and recv OP. The send and recv OP will be
+automatically added to
+the
+[`ProgramDesc`](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/program.md#blockdesc-and-programdesc) by
+the
+[distributed transpiler](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/dist_refactor/distributed_architecture.md#distributed-transpiler) when
+a local training `ProgramDesc` is transpiled to the distributed
+training `ProgramDesc`.
+
+
+## Send OP
+
+The send OP sends its input tensors to one or multiple recv OPs and
+then fetches its output tensors from the recv OPs.
+
+|Input|Description|
+|------|------|
+|X|Input tensor to be sent|
+
+|Output|Description|
+|------|------|
+|Out|Output tensor to be received from the recv OP|
+
+|Attribute|Description|
+|------|------|
+|endpoints|Endpoints to send the variables to|
```

Comment:
Would this be a list of OPs or just a list of receive OP endpoints ?

Reply:
It's a list of receive OP endpoints.

Reply:
And receive OP endpoints is basically the parameter_server endpoints, right ?

Reply:
Yes, they are the same.